src/testRunner/unittests/tsc/runWithoutArgs.ts

```diff
@@ -0,0 +1,10 @@
+namespace ts {
+    describe("unittests:: tsc:: runWithoutArgs::", () => {
+        verifyTsc({
+            scenario: "runWithoutArgs",
+            subScenario: "show help with ExitStatus.DiagnosticsPresent_OutputsSkipped",
+            fs: () => loadProjectFromDisk("tests/projects/runWithoutArgs"),

```

Comment:
You want to use `loadProjectFromFiles({})` instead.

Reply:
@sheetalkamat oke, thanks.